### PR TITLE
Adds gatsby integration package

### PR DESCRIPTION
**Pull request by WoLfulus**
This is a small integration package for v9 and gatsby using GraphQL endpoint instead.

### CODE REVIEW

**Comment by WoLfulus** [comment]
we might want to way for #3313, #3337 and #3785

**Comment by danieljwestman** [comment]
This is a very handy package, would love to have in Directus core. Whats the current status of the PR?

File: packages/gatsby-source-graphql/gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+	invariant(
+		!Number.isNaN(refreshInterval),
+		`\`gatsby-source-directus\` requires a valid \`dev.refresh\` to be specified.\n` +
+			`can be either a number (seconds) or a string (5s, 1m, ...)`
+	);
+
+	const directus = new Directus(endpoints.base, {
+		auth: {
+			mode: 'json',
+			// workaround for https://github.com/directus/directus/issues/3785
```
Comment (rijkvanzanten) [inline packages/gatsby-source-graphql/gatsby-node.js:116]: Can we get rid of these workarounds now that those issues have been resolved?

Comment (WoLfulus) [reply packages/gatsby-source-graphql/gatsby-node.js]: Done!

**Review by rijkvanzanten — APPROVED** [review]

**Comment by danielbidala** [comment]
@rijkvanzanten I get the following error when run `npm install`

npm ERR! Could not install from "packages\gatsby-source-directus" as it does not contain a package.json file.

**Comment by rijkvanzanten** [comment]
Ah yeah good catch @danielbidala  👍🏻 

@WoLfulus you accidentally named the folder `gatsby-source-graphql` instead of `graphql-source-directus`, and I totally missed that when reviewing the code ʕ•ᴥ•ʔ 

CleanShot 2021-02-08 at 13 35 56@2x

**Comment by danielbidala** [comment]
@rijkvanzanten `gatsby-source-directus` will be the right folder name :)

**Comment by rijkvanzanten** [comment]
Yup!

https://github.com/directus/directus/commit/d200c52e4f52181e7b2dfb47ef31a6f63aa2abb1

**Comment by WoLfulus** [comment]
🤦‍♂️

**Pull request merged**